src/Resources/config/services.yaml

```diff
@@ -35,3 +35,7 @@ services:
     MonsieurBiz\SyliusCmsPagePlugin\Routing\RequestContext:
         decorates: router.request_context
         arguments: ['@MonsieurBiz\SyliusCmsPagePlugin\Routing\RequestContext.inner']
+
+    Sylius\Calendar\Provider\DateTimeProviderInterface:
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏WDYTOT ? Else we can have a `DateTimeProviderInterface` in our namespace